time/src/testFixtures/kotlin/com/avito/time/FakeTimeProvider.kt

```diff
@@ -0,0 +1,20 @@
+package com.avito.time
+
+import java.util.Date
+
+class FakeTimeProvider : TimeProvider {
```

Comment:
Do we need a default constructor with exact time for testing?

Reply:
not until you asked somehow :)